lib/AbstractHandle/Utils/MongoUtil.py

```diff
@@ -81,10 +81,11 @@ def get_hid_counter(self):
         """
         get current handle id counter
         """
-        counter = self.hid_counter_collection.find({'_id': {'$eq': self._HID_COUNTER_ID}})
+        cursor = self.hid_counter_collection.find({'_id': {'$eq': self._HID_COUNTER_ID}})
+        counter = list(cursor)
 
-        if counter.count():
-            return counter.next().get('hid_counter')
+        if len(counter):
+            return counter[0].get('hid_counter')
         else:
             return 0
```

Comment:
If you change the find() to find_one() you'll get either the doc or None back which is simpler.

Reply:
👍

Reply:
:+1: